building/tracks/docs.md

```diff
@@ -10,7 +10,8 @@ docs
 ├── SNIPPET.txt (required)
 ├── REPRESENTER_NORMALIZATIONS.md (optional)
 ├── RESOURCES.md (required)
-└── TESTS.md (required)
+├── TESTS.md (required)
+└── config.json (required)
```

Comment:
Technically, this file is not required (as in: nothing breaks when it isn't there). I'd be happy to list it here as required though, as it is usually what one wants.

Reply:
Just made it optional! If it's optional, it's optional!

Reply:
Is it the `TESTS.md` that's optional or `config.json`?

Reply:
@thinkverse The whole reason this PR was started in the first place was because [V's config.json was missing, causing breakage](https://forum.exercism.org/t/v-missing-track-specific-documentation/3402) so I'm guessing `TESTS.md`